File: analysis/load.dart
import 'dart:convert';
import 'dart:io';

void main() {
  var repos = <String, Map<String, dynamic>>{};

  for (var repo in Directory("repositories").listSync()) {
    var issues = repos[repo.path] = {};
    for (var issue in Directory("${repo.path}/issues").listSync()) {
      issues[issue.path] = jsonDecode(File(issue.path).readAsStringSync());
    }
    for (var pull in Directory("${repo.path}/pulls").listSync()) {
      issues[pull.path] = jsonDecode(File(pull.path).readAsStringSync());
    }
  }

  for (var repo in repos.keys.toList()..sort()) {
    print("$repo: ${repos[repo]!.length}");
  }
}

File: analysis/load.pl
use strict;
use warnings;
use JSON::XS;
use File::Slurper 'read_binary';

my %repos;

for my $repo (glob "repositories/*") {
    my %issues;
    for my $issue (glob "$repo/issues/*") {
        $issues{$issue} = decode_json(read_binary($issue));
    }
    for my $pull (glob "$repo/pulls/*") {
        $issues{$pull} = decode_json(read_binary($pull));
    }
    $repos{$repo} = \%issues;
}

for my $repo (sort keys %repos) {
    my $count = keys %{$repos{$repo}};
    print "$repo: $count\n";
}
